docs/advanced_topics/customisation/page_editing_interface.md

```diff
@@ -52,7 +52,7 @@ class FundingPage(Page):
     ])
 ```
 
-For more details on how to work with `Panel`s and `PanelGroup`, see [](forms_panels_overview).
+For more details on how to work with `Panels` and `PanelGroup`, see [](forms_panels_overview).
```

Comment:
The current version of this is correct, `Panel` should be in backticks, and the pluralised (a) should be outside it. 

This is because `Panel` is a specific class name in the code and `Panels` is not.

Maybe another way to write this could be.

```suggestion
For more details on how to work with `Panel` and `PanelGroup classes,, see [](forms_panels_overview).
```